### DISCUSSION TITLE
Is er een studentenweergave mogelijk in Dodona?

### DISCUSSION THREAD

**Discussion by jonathanvynck** [dodona-edu/feedback-and-support#56] [Questions - Vragen] [answered]
Ik zou als leerkracht graag snel kunnen zien hoe Dodona er uitziet voor mijn leerlingen.
Zo kan ik de Safe Exam Browser uittesten voor het afleggen van een examen op Dodona.

**Comment by pdawyndt** [comment]
Dit is hoe je als lesgever kunt zien hoe de homepagina van een leerling uit je cursus eruitziet:

1. Klik in de navigatiebalk op het "Ingeschreven gebruikers"-icoontje

   ![image](https://github.com/user-attachments/assets/a5f20ecf-7419-4cd0-834f-b170c694dc30)

2. Klik op de naam van een gebruiker in de lijst van gebruikers

   ![image](https://github.com/user-attachments/assets/932fa8d7-addf-46c9-959e-3c377e077ded)

Dan zie je de homepagina van de cursus zoals die gebruiker die ziet.

**Comment by pdawyndt** [comment, marked as answer]
Voor examens of toesten kan je best de reeks zo instellen dat die enkel zichtbaar is via de geheime link.

1. Kies voor "Reeks bewerken" uit het menu van de reeks (drie puntje rechts onderaan het paneel voor de reeks)
   
   ![image](https://github.com/user-attachments/assets/16238e9f-a529-4f10-9536-3f913882e28e)

2. Stel de zichtbaarheid van de reeks in op "Enkel zichtbaar via geheime link"

   ![image](https://github.com/user-attachments/assets/d06120a0-bd30-45e7-8c87-9bca62daf5ec)

   Voor het examen of de toets kan je de zichtbaarheid van de reeks op "Verborgen voor studenten", en vlak voor het examen kan je die dan instellen op "Enkel zichtbaar via geheime link". Dan ben je zeker dat de reeks enkel toegankelijk is tijdens het examen of de toets.

   ![image](https://github.com/user-attachments/assets/f710d7f7-4fdd-4c13-96a4-de71f5054013)

   Daardoor is de vorige geheime link (die je al met de leerlingen van een andere klas gedeeld hebt) niet meer bruikbaar om toegang te krijgen tot de reeks.

3. Onder "Geheime links en tokens" een beetje verder op de pagina vind je een token-link die je met je student kunt delen. Enkel met die link is de reeks zichtbaar tijdens de toets of het examen.

   ![image](https://github.com/user-attachments/assets/47e573cc-49af-4fc4-b2b7-bb390806e2e9)

  Na de toets of het examen kan je de reeks terug op "Verborgen voor studenten" zetten. Als je na een toets of een examen de reeks nog eens wil hergebruiken voor een andere klas, dan je een nieuwe geheime link genereren voor die nieuwe klas door op de knop na de geheime link te klikken (enkel bruikbaar als je de reeks terug op "Enkel zichtbaar via geheime link" zet.

### Opmerkingen

1. Als lesgever krijg je uiteraard wel de gesloten of verborgen reeksen te zien, maar voor de veiligheid wordt alle informatie daarin vervaagd (die is enkel zichtbaar als je er met de muis over gaat) als je anonieme modus inschakelt. Op die manier krijgen leerlingen niet per ongeluk de info uit gesloten of verborgen reeksen te zien als je je cursus in de klas zou projecteren. Dat is ook de reden waarom de info in de reeks hierboven vervaagd weergegeven wordt.

   ![image](https://github.com/user-attachments/assets/f3021386-87b8-459a-89c3-f102ead788d2)

2. Als je als lesgever de geheime link gebruikt, dan zie je net zoals je leerlingen een pagina die enkel de reeks weergeeft, ook al zit die nog steeds in je cursus. Dat laatste kan je zien aan het feit dat de navigatiebalk nog steeds naar de cursus verwijst. **Er is vraag om op deze pagina voor studenten/leerlingen alle verwijzingen naar de homepagina van Dodona (linksboven in de navigatiebalk) en naar de cursus (in de navigatiebalk) weg te halen, zodat ze bij gebruik van de Safe Exam Browser niet aan hun oplossingen kunnen die ze op Dodona ingediend hebben. Deze feature is momenteel echter nog niet geïmplementeerd (opvolgen via #6016).**
